js/weather.js

```diff
@@ -1,35 +1,91 @@
 $(function () {
+  let selectedBox = 'current';
+  let apiData = {};
+
+  $('.toggle-buttons').click(function (e) {
+    // prevent checked box to be unchecked on click
+    if (!$(this).is(':checked')) {
+      e.preventDefault;
+      return false;
+    }
+    $(this).siblings().attr('checked', false); // uncheck checked siblings
```

Comment:
Looks like you've implemented checkboxes instead of buttons. This code seems to be ensuring that only one is checked at a time. If you use a radio group, you get this functionality built in with no need for custom code. 

For phase 1, can you implement this as two buttons instead of another element?